src/header/header.scss

```diff
@@ -304,7 +305,28 @@
   }
 
   &.iui-active {
```

Comment:
@FlyersPh9 @mayank99 We either make these changes or add active state to split button.
For me these changes seem better as they are small and easy to understand.

Reply:
Isn't there some mixin we can use?

Reply:
Oh, there isn't. In that case, I think we should extract out the active state logic (in `button/default.scss`) into a mixin that we can just include here.

After that, we can override anything if needed.